src/notifier/mqtt_notifier.py

```diff
@@ -30,9 +33,9 @@ def _set_config(self, config: dict):
         :param config: The YAML config for this notifier
         :returns: None
         """
-        self._topic = config["topic"]
-        self._qos: int = config.get("qos", 0)
-        self._retain: bool = config.get("retain", False)
```

Comment:
I’m not making an exhaustive list of mismatches but here I also noticed `qos` used to be 0, and it’s 1 now.

Reply:
Yeah, so what we have here is that I took the default values from config-example.yaml instead of from the code, with the assumption that anyone actually using it was likely always defaulting on the example instead of the default in the code.

So I think what we have here is a problem of having had two defaults and we need to collapse on only one. You wana do that one by one or should I just flip over from example config values to code defaults?